TESTING.md

```diff
@@ -143,3 +145,29 @@ Usage of /var/folders/35/r4w8sbqj2md1sg944kpnzyth0000gn/T/go-build3644709196/b00
   -var value
         Define a package var (k=v), with values being YAML-encoded; can be specified more than once
 ```
+
+### Building an Elastic Agent container image with a locally built APM Server
+
+It's possible to run `runapm` (as pictured above) and re-use the image that `runapm` builds to use in docker-compose
+files or run in ECE / ESS. However, it's also possible to only build a docker image without requiring the docker-compose
+project containers to be up and running with `systemtest/cmd/buildapm`.
+
+`buildapm` reads the `docker-compose.yml` at the root of the repository and uses that information to build an Elastic Agent
+docker image with an APM Server bundled that contains any local changes you might have made.
+
+By default, the same architecture (or platform in Docker lingo) as your local machine will be used. This may not be ideal
+if you have an architecture different than `amd64`, so you can specify the `-arch` flag.
```

Comment:
would it be better to have the default be `amd64`, since that's probably going to be the correct arch 99% of the time? How is this handled with docker for mac? I remember it being a virtualized amd64 linux machine, but that could (and probably is) be incorrect.

Reply:
I'm not it that since it's probably a safe assumption. I just defaulted to using whatever `runtime.GOARCH` is set to, but thinking about it now, `amd64` is way more widespread than `arm64`, so it sounds like a better default for now, until  (if) `arm64` takes over as the default architecture.

Reply:
addressed